packages/compass-shell/package.json

```diff
@@ -93,6 +95,7 @@
     "react-dom": "^17.0.2",
     "react-redux": "^8.0.5",
     "redux": "^4.2.1",
-    "sinon": "^9.2.3"
+    "sinon": "^9.2.3",
+    "zod": "^3.22.2"
```

Comment:
Do you think it would make sense to re-export this from user-data or incorporate into the constructor interface? Every package depending on user-data will need this dep and keep it up to date with the user-data package